feature/feature-home/src/main/java/com/comit/feature_home/mvi/WriteScheduleContract.kt

```diff
@@ -16,5 +16,9 @@ sealed class WriteScheduleSideInEffect {
 
     object WriteScheduleSuccess : WriteScheduleSideInEffect()
 
-    object WriteScheduleFail : WriteScheduleSideInEffect()
+    object InputTextFormError : WriteScheduleSideInEffect()
+
+    object TokenException : WriteScheduleSideInEffect()
```

Comment:
Event는 동작을 의미하면 좋을 것 같습니다.

```suggestion
    object NeedLogin : WriteScheduleSideInEffect()
```